cmd/dusk/server.go

```diff
@@ -244,16 +239,26 @@ func Setup() *Server {
 	// Creating the peer factory
 	readerFactory := peer.NewReaderFactory(processor)
 
+	// Create the listener and contact the voucher seeder
+	gossip := protocol.NewGossip(protocol.TestNet)
+	connector := peer.NewConnector(eventBus, gossip, cfg.Get().Network.Port, processor, protocol.ServiceFlag(cfg.Get().Network.ServiceFlag), peer.Create)
+
+	seeders := cfg.Get().Network.Seeder.Addresses
+	for _, seeder := range seeders {
+		if err = connector.Connect(seeder); err != nil {
```

Comment:
If a node fails to connect to all vouchers here we should at least panic for now. Later on, we can implement a reattempt-to-connect procedure.

Reply:
Agreed. Will extrapolate into an issue